tool/jt.rb

```diff
@@ -511,7 +511,9 @@ def build(*options)
         mx 'build', '--dependencies', 'SULONG'
       end
     when "cexts" # Included in 'mx build' but useful to recompile just that part
-      raw_sh "make", chdir: "#{TRUFFLERUBY_DIR}/src/main/c"
+      require 'etc'
+      cores = Etc.respond_to?(:nprocessors) ? Etc.nprocessors : 4
```

Comment:
I wouldn't mind leaving 1 or 2 cores free so I can do other work. But we can see how this goes. The compilations are fairly short.

Reply:
Yeah it's all rather small C files, and typically lots of IO so we could even go above the number of cores without much load.